docs/rfc/rfc-009-consensus-parameter-upgrades.md

```diff
@@ -72,6 +72,14 @@ parameters but does not verify that they are the same across all validators by
 referencing them in the hash. This seems reasonably safe given the fact that
 only a very far subset of the consensus parameters are currently verified at all.
 
+#### Version The Consensus Parameter Hash Scheme
+
+The upcoming work on [soft upgrades](https://github.com/tendermint/spec/pull/222)
+proposes applying different hashing rules depending on the active block version.
+The consensus parameter hash could be versioned in the same way. When different
+block versions are used, a different set of consensus parameters will be included
+in the hash.
```

Comment:
This is compatible with the original "defaults" based proposal, in that we still need some rule for how to bootstrap new consensus parameters. 👍🏼